.eslintcache

```diff
@@ -1 +1 @@
\ No newline at end of file
+[{"/home/barrod/Git/equipo-4/src/components/App.js":"1","/home/barrod/Git/equipo-4/src/components/About.jsx":"2","/home/barrod/Git/equipo-4/src/components/Header.jsx":"3","/home/barrod/Git/equipo-4/src/components/Home.jsx":"4"},{"size":530,"mtime":1606857650704,"results":"5","hashOfConfig":"6"},{"size":160,"mtime":1606857650704,"results":"7","hashOfConfig":"6"},{"size":1219,"mtime":1606857650704,"results":"8","hashOfConfig":"6"},{"size":281,"mtime":1606859268887,"results":"9","hashOfConfig":"6"},{"filePath":"10","messages":"11","errorCount":0,"warningCount":1,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},"hpiodj",{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"14","messages":"15","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"16","messages":"17","errorCount":0,"warningCount":5,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},"/home/barrod/Git/equipo-4/src/components/App.js",["18"],"/home/barrod/Git/equipo-4/src/components/About.jsx",[],"/home/barrod/Git/equipo-4/src/components/Header.jsx",[],"/home/barrod/Git/equipo-4/src/components/Home.jsx",["19","20","21","22","23"],{"ruleId":"24","severity":1,"message":"25","line":10,"column":5,"nodeType":"26","messageId":"27","endLine":10,"endColumn":9},{"ruleId":"24","severity":1,"message":"28","line":1,"column":17,"nodeType":"26","messageId":"27","endLine":1,"endColumn":25},{"ruleId":"24","severity":1,"message":"29","line":2,"column":10,"nodeType":"26","messageId":"27","endLine":2,"endColumn":14},{"ruleId":"24","severity":1,"message":"30","line":2,"column":16,"nodeType":"26","messageId":"27","endLine":2,"endColumn":19},{"ruleId":"24","severity":1,"message":"31","line":2,"column":21,"nodeType":"26","messageId":"27","endLine":2,"endColumn":27},{"ruleId":"24","severity":1,"message":"32","line":2,"column":29,"nodeType":"26","messageId":"27","endLine":2,"endColumn":36},"no-unused-vars","'Link' is defined but never used.","Identifier","unusedVar","'useState' is defined but never used.","'Flex' is defined but never used.","'Box' is defined but never used.","'Button' is defined but never used.","'Heading' is defined but never used."]
```

Comment:
![image](https://user-images.githubusercontent.com/12115092/100882473-02afad00-348e-11eb-8c9e-8cb85a0802aa.png)

Reply:
It shall be slain by means of gitignore next time!